apps/src/templates/policy_compliance/ParentalPermissionModal/index.story.tsx

```diff
@@ -33,18 +33,22 @@ const useReducerStub = sinon.stub(React, 'useReducer');
 const Template = (state: object = {}) => {
   useReducerStub.returns([state, spy]);
 
-  return <ParentalPermissionModal lockoutDate={new Date()} />;
+  return <ParentalPermissionModal lockoutDate={Date()} />;
 };
 
 export const NewRequestForm = () => {
-  return Template();
+  const state = {
+    parentalPermissionRequest: null,
+  };
+
+  return Template(state);
 };
 
 export const UpdateRequestForm = () => {
   const state = {
     parentalPermissionRequest: {
       parent_email: 'parent@email.com',
-      requested_at: new Date().toISOString(),
+      requested_at: Date(),
```

Comment:
Seems so haphazard to trust the localized string of the date instead of something more consistent. Is this really the way one is supposed to handle dates? I'm sure `moment` can magically handle anything but it feels so wrong.

Reply:
I'll revert this chnage

Reply:
I appreciate your effort on this. I'm never quite confident about handling date serialization. I normally default to the iso format whenever there is doubt, but I'm still ready to be convinced otherwise.